frontend/src/components/Login.js

```diff
@@ -1,45 +1,114 @@
 import React, {Component} from 'react';
-import {Button, Checkbox, Grid, Form, Header, Segment, Message} from "semantic-ui-react";
-import {Link} from "react-router-dom";
+import {Button, Grid, Form, Header, Segment, Message} from "semantic-ui-react";
+import {Link, Redirect} from "react-router-dom";
+import axios from 'axios';
 
 export class Login extends Component {
 
-    constructor() {
-        super();
-        this.onClickHandler = this.onClickHandler.bind(this);
-    }
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: '',
+            redirect: false,
+        };
 
-    onClickHandler(e) {
-        alert('Request sent, trying to authenticate');
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
     }
 
+    /*
+    Source:
+    https://stackoverflow.com/questions/44072750/how-to-send-basic-auth-with-axios
+     */
+    handleSubmit = event => {
+        event.preventDefault();
+
+        const username = this.state.username;
+        const password = this.state.password;
+        axios.post('http://127.0.0.1:5000/login', {}, {
+            auth: {
+                username: username,
+                password: password,
+            }
+        }).then(res => {
+            //Setting values into sessionStorage
+            sessionStorage.setItem("token", res.data.token);
+            sessionStorage.setItem("username", res.data.user.username);
+            sessionStorage.setItem("publicid", res.data.user.public_id);
```

Comment:
Store de hele user (`res.data.user`) maar in localStorage. Anders moet je deze code steeds aanpassen als de user meer velden krijgt.